main.py

```diff
@@ -0,0 +1,25 @@
+while True:
+    i = input("input temp with type (f/c/k)>")
+    if i == "exit" :
+        break
+    t = i.split("f")
+    if len(t) == 1 :
+        t = i.split("c")
+        if len(t) == 1:
+            t = i.split("k")
```

Comment:
You might have been able to simplify this with a regular expression or string parsing